esp/src/src-react/components/Queries.tsx

```diff
@@ -25,18 +26,12 @@ const FilterFields: Fields = {
 };
 
 function formatQuery(filter: any, wuid?: string) {
-    if (wuid !== undefined) {
-        return {
-            WUID: wuid
-        };
-    }
-    const retVal = { ...filter };
-    if (filter.StartDate) {
-        retVal.StartDate = new Date(filter.StartDate).toISOString();
-    }
-    if (filter.EndDate) {
-        retVal.EndDate = new Date(filter.StartDate).toISOString();
-    }
+    const retVal = {
+        ...filter,
+        PriorityLow: filter.Priority,
+        PriorityHigh: filter.Priority
```

Comment:
Were these supposed to be different values or anything?

Reply:
It is correct - we only filter on a single priority and not a range of priorities (confusing for the user).